n_data : 3
goal_pos_x : [-2.88393,0.0535736,2.71558]
goal_pos_y : [-1.56046,0.0853193,2.8574]
goal_pos_z : [0,0,0]
goal_ori_x : [0,0,0]
goal_ori_y : [0,0,0]
goal_ori_z : [0.0288123,0.381756,0.361229]
goal_ori_w : [0.999585,0.924263,0.932477]
